spec/std/http/cookie_spec.cr

```diff
@@ -146,36 +146,80 @@ module HTTP
 
       it "parse max-age as seconds from Time.now" do
         cookie = parse_set_cookie("a=1; max-age=10")
-        delta = cookie.expires.not_nil! - Time.now
-        delta.should be > 9.seconds
-        delta.should be < 11.seconds
+        cookie.max_age.should eq 10.seconds
 
         cookie = parse_set_cookie("a=1; max-age=0")
-        delta = Time.now - cookie.expires.not_nil!
-        delta.should be > 0.seconds
-        delta.should be < 1.seconds
+        cookie.max_age.should eq 0.seconds
+      end
+    end
+
+    describe "expiration_time" do
+      it "sets expiration_time to be current when max-age=0" do
+        cookie = parse_set_cookie("bla=1; max-age=0")
+        (Time.now - cookie.expiration_time.not_nil!).should be.<= 1.seconds
```

Comment:
I'm really sorry that I my suggestion was a bit inprecise. 😢  I don't know why I assumed the period to be necessary...

`be <= 1.seconds` works as well and reads much better.

Reply:
Will fix. No problem; I could have looked through more docs myself.